src/writeData/subscriptions/components/CreateSubscriptionPage.tsx

```diff
@@ -57,7 +60,7 @@ const CreateSubscriptionPage: FC = () => {
   const {formContent, saveForm, updateForm, loading} = useContext(
     SubscriptionCreateContext
   )
-  const quartzMe = useSelector(getQuartzMe)
+  const identity = useSelector(selectCurrentIdentity)
   const buckets = useSelector((state: AppState) =>
```

Comment:
This is a cloud-only component, so there is no need for `CLOUD` guards.

The path is: 
- `SetOrg` contains a `CLOUD` and `isFlagEnabled('subscriptionsUI')` check for whether the `CreateSubScriptionForm` can be loaded.
- `CreateSubscriptionForm` is actually a lazy import of `CreateSubscriptionPage`
- `CreateSubscriptionPage` is a default export parent component consisting of several components from `CreateSubcriptionPage`

There are no other references in the UI to `CreateSubscriptionPage`